include/cspublic.h

```diff
@@ -248,6 +248,8 @@ enum
 #define CS_SEC_APPDEFINED CS_SEC_APPDEFINED
 	CS_STICKY_BINDS = 9151,
 #define CS_STICKY_BINDS CS_STICKY_BINDS
+	CS_CLIENTCHARSET = 9152,
```

Comment:
I think would be better as 9300, Sybase is using 9152 for CS_HAVE_CMD

Reply:
Sure thing, thanks for the quick reply :)